docker-compose.yaml

```diff
@@ -0,0 +1,24 @@
+version: "3.7"
+
+services:
+  mysql:
+    build: 
+      context: ./sql
+    restart: unless-stopped
+    volumes:
+      - liveqa-mysql-data:/var/lib/mysql
+    env_file:
+      - .env
+
+  php:
+    build: 
+      context: ./
+    restart: unless-stopped
+    ports:
+      - 22125:80
```

Comment:
Is there a reason for this number? Maybe we should use for default deployment files on docker and podman the same port.